nilearn/plotting/img_plotting.py

```diff
@@ -208,6 +208,11 @@ class _MNI152Template(object):
     vmax   = None
     _shape  = None
 
+    def __init__(self, *args, **kwargs):
+        # This looks like a hack but we want a constant image compatible with
+        # all interfaces
+        pass
```

Comment:
Sorry I don't get this, the only use I could find of using _MNI152Template was without any arguments in the `__init__`.

Also just curious, what's the advantage of having _MNI152Template be an image?